src/bridge.ts

```diff
@@ -85,7 +85,7 @@ export interface BridgeController {
      * not supplied, no rooms will be provisioned on alias queries. Provisioned rooms
      * will automatically be stored in the associated `roomStore`. */
     onAliasQuery?: (alias: string, aliasLocalpart: string) =>
-        PossiblePromise<{creationOpts: Record<string, unknown>, remote?: RemoteRoom}|null|void>;
+        PossiblePromise<{roomId?: string, creationOpts: Record<string, unknown>, remote?: RemoteRoom}|null|void>;
```

Comment:
You are no longer required to pass these in:

```suggestion
        PossiblePromise<{roomId?: string, creationOpts?: Record<string, unknown>, remote?: RemoteRoom}|null|void>;
```

Reply:
Great spot! :+1: